tests/test_api.py

```diff
@@ -1925,7 +1925,7 @@ def test_config_map(self, osbs, name, data):
             'apiVersion': 'v1',
             'kind': 'ConfigMap',
             'metadata': {'name': name},
-            'data': data,
+            'data': {key: json.dumps(value) for key, value in data.items()},
```

Comment:
Thanks for clarifying this, but I get travis build failures against __builtin__ when I do this.